comp/otelcol/ddflareextension/impl/extension.go

```diff
@@ -177,19 +183,27 @@ func (ext *ddExtension) Shutdown(ctx context.Context) error {
 
 // ServeHTTP the request handler for the extension.
 func (ext *ddExtension) ServeHTTP(w http.ResponseWriter, _ *http.Request) {
-	customer, err := ext.configStore.getProvidedConfAsString()
-	if err != nil {
-		w.WriteHeader(http.StatusInternalServerError)
-		fmt.Fprintf(w, "Unable to get provided config\n")
-		return
+	var (
+		customer string
+		err      error
+	)
+	if ext.configStore.isProvidedConfigSupported() {
```

Comment:
I don't see a need for is ProvidedConfig supporter. Lets get 	`customer, err = ext.configStore.getProvidedConfAsString()` , as long as it isn't empty string we have provided config